odoo/addons/base/data/res_currency_data.xml

```diff
@@ -36,7 +36,8 @@
         <record id="CHF" model="res.currency">
             <field name="name">CHF</field>
             <field name="full_name">Swiss franc</field>
-            <field name="symbol">CHF</field>
+            <field name="symbol">fr.</field>
```

Comment:
Is it? Did they recently changed that? Just curious seing this diff :)
Went through a ~10 of swiss eshop and ~10 swiss invoice and never saw fr., always CHF?

Reply:
It seems SFr., Fr. and CHf are all valid, maybe we just stick to CHF (as it is most international)